### DISCUSSION TITLE
Building on Fedora 36

### DISCUSSION THREAD

**Discussion by brandonk10** [OpenIDC/mod_auth_openidc#925] [Q&A] [answered]
I have been building the module from source, and I recently upgraded to Fedora 36, and started getting this build error:

```
/usr/bin/ld: test/test-cmd.o: relocation R_X86_64_32 against `.rodata.str1.1' can not be used when making a PIE object; recompile with -fPIE
/usr/bin/ld: test/stub.o: relocation R_X86_64_32 against `.rodata.str1.1' can not be used when making a PIE object; recompile with -fPIE

```
I'm not really sure what drove the change, but I cant find any recent commits that look like they are related, so my best guess is an updated toolchain.

I can get it to compile/run by adding either -fPIC or -fPIE to CFLAGS, but I'm not sure if this is right way to solve this.  I do see -fPIC in libtool.

**Comment by zandbelt** [comment, marked as answer]
added `-fPIC` in 39c5c83be4a3af6a202c39e195de267a11910bb6, thanks

**Reply by brandonk10** [reply]
Confirmed, thanks!
